packages/cactus-plugin-ledger-connector-fabric/tsconfig.json

```diff
@@ -13,7 +13,10 @@
     "src/**/*.json"
   ],
   "exclude": [
-    "./src/test/typescript/fixtures/go/basic-asset-transfer/chaincode-typescript/**/*.ts"
+    "./src/test/typescript/fixtures/go/basic-asset-transfer/chaincode-typescript/**/*.ts",
+    "./src/test/typescript/fixtures/go/lock/chaincode-typescript/src/*.ts",
```

Comment:
@jscode017 Are these exclusions still needd/effective?

Reply:
Yeah, I think this one: ./src/test/typescript/fixtures/go/basic-asset-transfer/chaincode-typescript/**/*.ts
is still needed, since the contract use different settings